qiita_pet/templates/private_studies.html

```diff
@@ -44,13 +44,16 @@
             $("#users_list").trigger("chosen:updated");
 
         });
-};
+}
 
+function fillabstract(table, row) {
```

Comment:
This function is being used in two different templates, would you mind centralizing it somewhere? Perhaps qiita.js? And change the name from fillabstract->fillAbstract.